packages/rln/src/identity.ts

```diff
@@ -28,4 +28,13 @@ export class IdentityCredential {
       idCommitmentBigInt
     );
   }
+
+  public toJSON(): Record<string, number[]> {
```

Comment:
`toJSON` must return JSON string (aka `JSON.stringify`) type per what is common in JS
example - https://developer.mozilla.org/en-US/docs/Web/API/URL/toJSON